src/Link.js

```diff
@@ -4,8 +4,19 @@ export function Link(props, children) {
   props.href = props.to
   props.to = null
 
-  props.onclick = function(element) {
-    element.preventDefault()
+  props.onclick = function(event) {
+    if (
+      event.button !== 0 ||
```

Comment:
@SkaterDad Since we are not (currently) let the user overwrite the `onclick` handler, I think it makes no sense to add [<samp>e.defaultPrevented</samp>](https://developer.mozilla.org/en-US/docs/Web/API/Event/defaultPrevented) either. What do you think?